spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -215,8 +220,10 @@ def create!(name: nil, bundle_id: nil, certificate: nil, devices: [], mac: false
           if devices.nil? or devices.count == 0
             if self == Development or self == AdHoc
               # For Development and AdHoc we usually want all compatible devices by default
-              if mac
+              if platform == 'mac'
```

Comment:
Ah, I think it wants you to prefer:

```ruby
devices = if platform == 'mac'
            Spaceship::Device.all_macs
          elsif platform == 'ios' and !sub_platform.nil? and sub_platform.casecmp('tvos') == 0
            Spaceship::Device.all_apple_tvs
          else
          	Spaceship::Device.all_for_profile_type(self.type)
          end
```

but feel free to ignore that suggestion for now!